nx.json

```diff
@@ -112,7 +112,59 @@
       "unitTestRunner": "vitest"
     },
     "@nx/angular:component": {
-      "style": "css"
+      "style": "css",
+      "type": "ng"
+    },
+    "@schematics/angular:component": {
+      "type": "ng"
```

Comment:
where are these coming from? Does that mean we are generating with the `ng` extension?

Reply:
the migration generated "component" and I've changed to "ng"
btw, do we really need these? I don't use generators.

Reply:
Me neither, we can remove them. Someone asked me already, if testronaut requires the ng extension because he saw some of our code.

Reply:
I'll remove the schematic configs right away.